src/io.cpp

```diff
@@ -0,0 +1,81 @@
+#include <s2/s1angle.h>
+#include <s2geography.h>
+
+#include "constants.hpp"
+#include "geography.hpp"
+#include "pybind11.hpp"
+
+namespace py = pybind11;
+namespace s2geog = s2geography;
+using namespace spherely;
+
+PyObjectGeography from_wkt(py::str a, bool oriented, bool planar) {
+#if defined(S2GEOGRAPHY_VERSION_MAJOR) && \
+    (S2GEOGRAPHY_VERSION_MAJOR >= 1 || S2GEOGRAPHY_VERSION_MINOR >= 2)
+    s2geog::geoarrow::ImportOptions options;
+    options.set_oriented(oriented);
+    if (planar) {
+        auto tol = S1Angle::Radians(100.0 / EARTH_RADIUS_METERS);
+        options.set_tessellate_tolerance(tol);
+    }
+    s2geog::WKTReader reader(options);
+#else
+    if (planar || oriented) {
+        throw std::invalid_argument(
+            "planar and oriented options are only available with s2geography >= 0.2");
```

Comment:
```suggestion
            "planar and oriented options are only available with s2geography >= 1.2");
```

(per above `if`?)

Reply:
I _think_ I have it correct now.. In any case, the s2geography version is certainly 0.2 (https://github.com/paleolimbot/s2geography/blob/9ebd7b106ba2ae30046cb4f3e9561a8bbdb18b9b/CMakeLists.txt#L6).

I initially did the `#if` version check wrong, see https://github.com/benbovy/spherely/pull/50/commits/de19f0eb819b302f3a88d56d79e87f3b33b0c788 for correcting it. 

We want that both version of 0.2 or 1.1 result in True, with the current version being 0.1 returning False, so that means `major >= 1 OR minor >= 2`, I think?